### ISSUE TITLE
[Feature]: Want to make functional SIgnUp/login page 

### ISSUE THREAD

**Issue by nishant0708** [harmeetsingh11/RefactorMate#59]
### Description

want to make signup/login page and maintain their information in database.

### Justification

It will be useful because we can store info about refactor code that users had then. And user could see them in profile .just like chatgpt where our previous chat get saved

### Checklist

- [X] I have starred the repository
- [X] I have checked the existing [issues](https://github.com/harmeetsingh11/RefactorMate/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/harmeetsingh11/RefactorMate/blob/main/CONTRIBUTING.md)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/harmeetsingh11/RefactorMate/blob/main/CODE_OF_CONDUCT.md)
- [X] I am willing to work on this issue (optional)

**Comment by amanjaiswal7236** [comment]
Can you please assign this to me under gssoc??

**Comment by harmeetsingh11** [comment]
That sounds like a great idea. 
@nishant0708, I'm assigning this issue to you. Wish you all the best. Don't forget to star the repo. Happy contributing!

**Comment by nishant0708** [comment]
Thanks

**Comment by nishant0708** [comment]
hey @harmeetsingh11 
![image](https://github.com/harmeetsingh11/RefactorMate/assets/101548649/11c293c9-4e5e-454b-9d0c-3e2f4a65df56)
i am facing this issue what should i do??

**Comment by harmeetsingh11** [comment]
@nishant0708 Will look into this issue.

**Comment by nishant0708** [comment]
Hey @harmeetsingh11 got any solution?

**Comment by harmeetsingh11** [comment]
Hey @nishant0708, the issue has been resolved now. Apologies for the delay in fixing it. Follow these steps to ensure everything works smoothly:

1. **Re-clone the Repository:** 
   Clone the repository again to ensure you have the latest changes.

2. **Install pnpm:** 
   If you haven't already, make sure you have pnpm installed. You can do this by running:
   ```sh
   npm install -g pnpm
   ```

3. **Install Dependencies:** 
   Run the following command to install all dependencies using pnpm:
   ```sh
   pnpm install
   ```

4. **Run Frontend and Backend using the following commands:** 
   
   ```sh
   pnpm run frontend
   ```
   ```sh
   pnpm run backend
   ```

These steps should ensure that everything is set up correctly. Let me know if you need any further assistance.

**Comment by AdarshGT33** [comment]
Where are we in this issue? I can contribute towards building the backend services for signup and login

**Comment by harmeetsingh11** [comment]
@nishant0708 are you working on this issue?

**Comment by nishant0708** [comment]
yes @harmeetsingh11 i am facing issues like 
![image](https://github.com/harmeetsingh11/RefactorMate/assets/101548649/881727dd-df05-4c38-a095-d104140bfdb9)
i dont know where the error is so just figuring it out. hey @harmeetsingh11 just can you tell me if i have to install any library like cors we usually do npm so here i need to do pnpm i cors?? right??

**Comment by harmeetsingh11** [comment]
@nishant0708 cors package can be installed using `pnpm add cors`

**Comment by AritraDey-Dev** [comment]
i want to work on this issue under dwoc.Can you please assign me this under dwoc?

**Comment by AritraDey-Dev** [comment]
> yes [@harmeetsingh11](https://github.com/harmeetsingh11) i am facing issues like ![image](https://github.com/harmeetsingh11/RefactorMate/assets/101548649/881727dd-df05-4c38-a095-d104140bfdb9) i dont know where the error is so just figuring it out. hey [@harmeetsingh11](https://github.com/harmeetsingh11) just can you tell me if i have to install any library like cors we usually do npm so here i need to do pnpm i cors?? right??

@harmeetsingh11 this issue I am also facing while running it lo locally when I am adding new things in the file.

**Comment by harmeetsingh11** [comment]
@AritraDey-Dev I had already opened a issue regarding this #163

**Comment by AritraDey-Dev** [comment]
> [@AritraDey-Dev](https://github.com/AritraDey-Dev) I had already opened a issue regarding this [#163](https://github.com/harmeetsingh11/RefactorMate/issues/163)

yes because of this i stopped working on this issue.Once the parent issue is solved i will look into this.

**Comment by harmeetsingh11** [comment]
> > [@AritraDey-Dev](https://github.com/AritraDey-Dev) I had already opened a issue regarding this [#163](https://github.com/harmeetsingh11/RefactorMate/issues/163)
> 
> yes because of this i stopped working on this issue.Once the parent issue is solved i will look into this.

@AritraDey-Dev are you interested in solving the parent issue (#163)?
